src/system.cr

```diff
@@ -22,4 +22,14 @@ module System
   def self.cpu_count
     Crystal::System.cpu_count
   end
+
+  # Returns the short user name of the currently logged in user or `nil` if it
+  # can't be determined. Note that this information is not secure.
```

Comment:
Additionally, when can this return nil? I don't see a reason to declare it as nillable.